### PR TITLE
Consume claimable tokens local integration

### PR DESCRIPTION
**Pull request by hareeshnagaraj**
### Description

Enables local integration with PR found here - https://github.com/AudiusProject/audius-protocol/pull/1632

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.

Validated manually

### How will this change be monitored?

For features that are critical or could fail silently please describe the monitoring/alerting being added.

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/hn_consume_claimable_tokens_local

File: src/services/AudiusBackend.js [hunk 1]
```diff
@@ -385,8 +385,10 @@ class AudiusBackend {
     // initialize libs
     let libsError = null
     const { web3Error, web3Config } = await AudiusBackend.getWeb3Config()
+    console.log(web3Config)
```
Comment (hareeshnagaraj) [inline src/services/AudiusBackend.js:388]: remove this

**Review by hareeshnagaraj — COMMENTED** [review]
https://github.com/AudiusProject/audius-protocol/pull/1649#issuecomment-881128975

**Comment by hareeshnagaraj** [comment]
```
ssh -N -L 3000:127.0.0.1:3000 -i ~/.ssh/google_compute_engine ubuntu@34.71.153.101
```

File: src/services/AudiusBackend.js [hunk 2]
```diff
@@ -452,6 +452,7 @@ class AudiusBackend {
 
       AudiusBackend.sanityChecks(audiusLibs)
     } catch (err) {
+      console.log(err)
```
Comment (piazzatron) [inline src/services/AudiusBackend.js:455]: intentional?

**Review by piazzatron — APPROVED** [review]
LGTM but would like @raymondjacobson to take a look

File: src/services/AudiusBackend.js (hunk 2 above)
Comment (raymondjacobson) [reply src/services/AudiusBackend.js:455]: no harm, but should be console.error probs

**Review by raymondjacobson — APPROVED** [review]
Fire!!!

**Pull request merged**